crates/ruff_linter/src/rules/ruff/snapshots/ruff_linter__rules__ruff__tests__ruf100_3.snap

```diff
@@ -133,19 +134,20 @@ RUF100_3.py:7:10: RUF100 [*] Unused blanket `noqa` directive
 5 5 | print()  # noqa  # comment
 6 6 | print()  # noqa comment
 7   |-print()  # noqa  comment
-  7 |+print()  # comment
+  7 |+print()
 8 8 | print(a)  # noqa
 9 9 | print(a)  # noqa # comment
 10 10 | print(a)  # noqa  # comment
 
 RUF100_3.py:14:1: RUF100 [*] Unused `noqa` directive (unused: `E501`, `F821`)
    |
-12 | print(a)  # noqa  comment
-13 | 
-14 | # noqa: E501, F821
-   | ^^^^^^^^^^^^^^^^^^ RUF100
-15 | # noqa: E501, F821 # comment
-16 | print()  # noqa: E501, F821
+12 |   print(a)  # noqa  comment
+13 |   
+14 | / # noqa: E501, F821
+15 | | # noqa: E501, F821 # comment
+   | |_^ RUF100
```

Comment:
Not 100% sure why this range happens this way, but the fix is identical...

Reply:
Oh, I think I got it. The new range ends using the `edit.end()` instead of `directive.end()` and the former includes the newline character while the latter doesn't.